cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -13,7 +13,7 @@ class HttpCache
   # A map from script name to script level URL pattern.
   CACHED_SCRIPTS_MAP = {
     'starwars' => '/s/starwars/stage/1/puzzle/*',
```

Comment:
should we make this a bit less hardcoded to avoid errors like this in the future?
````
CACHED_SCRIPTS_MAP = {
# just the special cases like hourofcode
}
%w(starwars starwarsblocks mc frozen gumball).each do |script_name|
  CACHED_SCRIPTS_MAP[script_name] = "s/#{script_name}/stage/1/puzzle/*"
end
````
(you can totally add things to a constant hash, it's a hash * const not a const hash * (if I remember my c correctly))